llvm/include/llvm/Analysis/CtxProfAnalysis.h

```diff
@@ -18,6 +20,9 @@ namespace llvm {
 
 class CtxProfAnalysis;
 
+using CtxProfFlatProfile =
+    DenseMap<GlobalValue::GUID, SmallVector<uint64_t, 16>>;
```

Comment:
No strong data right now - we have data about the size of contexts during collection (they are all small), but this here would be after IPO. Frankly, we could have it set "1" because all contexts must have at least 1 counter, and then, because all contexts belonging to a function have the same size, there'll be at most one other heap allocation.